### ISSUE TITLE
Verbosity in scripts

### ISSUE THREAD

**Issue by dlakaplan** [nanograv/PINT#1377]
Currently executable scripts change logging level via:
```
(pintdev) kaplan@plock[~/pythonpackages/PINT_dlakaplan] (polycoreadwrite) % pintk -h
usage: pintk [-h] [--ephem EPHEM] [--test]
             [-f {auto,downhill,WLSFitter,GLSFitter,WidebandTOAFitter,PowellFitter,DownhillWLSFitter,DownhillGLSFitter,WidebandDownhillFitter,WidebandLMFitter}]
             [-v] [--log-level {TRACE,DEBUG,INFO,WARNING,ERROR}]
             parfile timfile
...
  --log-level {TRACE,DEBUG,INFO,WARNING,ERROR}
                        Logging level
```
This isn't necessarily the most standard interface, but it is explicit.  On the other hand we could easily change them all to:
```
--verbose
```
where the number of `--verbose` or `-v` arguments increase the verbosity.  Do people have opinions on which way to go?

**Comment by scottransom** [comment]
@dlakaplan We can have both, right?  If the default log level is, say, `INFO`, specifying `--log-level` gives you exactly what you might want.  But you could also specify `--verbose` and get the equivalent of one extra level of logging (i.e. `DEBUG`) whereas `--quiet` would give you `WARNING`.  And multiple uses of those just increment or decrement the level appropriately.  Basically they are all just different ways of changing the log level.

**Comment by dlakaplan** [comment]
Yes, they could both be there, but I feel that it might be confusing to read.  But if that's the consensus then I can do it.

**Comment by dlakaplan** [comment]
If you want both, is this what you are looking for?

```
(pintdev) kaplan@plock[~/pythonpackages/PINT_dlakaplan] (scriptverbose) % compare_parfiles -h                                                    
usage: compare_parfiles [-h] [--dmx] [--no-dmx] [--convertcoordinates] [--no-convertcoordinates] [--sigma SIGMA] [--uncertainty_ratio UNCERTAINTY_RATIO]
                        [--nocolor] [--comparison {max,med,min,check}] [--format {text,markdown}] [--log-level {TRACE,DEBUG,INFO,WARNING,ERROR,CRITICAL}] [-v] [-q]
                        input1 input2
...
  --log-level {TRACE,DEBUG,INFO,WARNING,ERROR,CRITICAL}
                        Logging level (default: WARNING)
  -v, --verbosity       Increase output verbosity (default: 0)
  -q, --quiet           Decrease output verbosity (default: 0)
```
So you can do `compare_parfiles --log-level=INFO`, and then any `-v` will make it more verbose, and `-q` will be less so.  I have a (slightly clumsy) implementation of this, so if people want this I can roll it out.

**Comment by scottransom** [comment]
Yeah, I think that is good -- I like it.  If you are a developer it seems like you might explicitly want to set the log-level.  But as a user, sometimes you just want less (or more) info and don't want to remember the various debug levels....

**Comment by dlakaplan** [comment]
Closed by #1380
